Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -333,6 +333,28 @@ extension ContentBlockerHelper {
 
 }
 
+@available(iOS 11.0, *)
+extension ContentBlockerHelper {
+
+    static func setTrackingProtectionMode(_ prefs: Prefs, tabManager: TabManager, enabled: Bool) {
+        guard let isPrivate = tabManager.selectedTab?.isPrivate else { return }
+        let key = isPrivate ? ContentBlockingConfig.Prefs.PrivateBrowsingEnabledKey : ContentBlockingConfig.Prefs.NormalBrowsingEnabledKey
+        prefs.setBool(enabled, forKey: key)
+        ContentBlockerHelper.prefsChanged()
+    }
+
+    static func isActivated(_ prefs: Prefs, tabManager: TabManager) -> Bool {
```

Comment:
prefer: `isTrackingProtectionActive(_ tabManager: tabManager)`
(prefs is unused BTW)
Consider adding a comment to explain that it checks the pref setting for the current privacy state and the per-tab setting. Helps the future reader with the considerations involved in what is a seemingly simple check.